storage_service/locations/views.py

```diff
@@ -44,8 +44,23 @@ def get_delete_context_dict(request, model, object_uuid, default_cancel='/'):
 
 def package_list(request):
     packages = Package.objects.all()
+
+    # Amalgate last fixity check date and result with package data
+    package_data = []
+    for package in packages:
+        fixity_checks = FixityLog.objects.filter(package=package).order_by('-datetime_reported')
+        latest_check = fixity_checks[0].datetime_reported if fixity_checks else None
+        latest_result = fixity_checks[0].success if fixity_checks else None
+        package_data.append((latest_check, latest_result, package))
```

Comment:
Since we need this info in several places (the package list, the Locations list where it shows several packages) then we probably want to generate this information in once place.  I think it makes sense to add latest_check & latest_result to the Package as @propeties.  Eg.

```python
class Package(models.Model):
    ...
    @property
    def latest_fixity_check(self):
        try:
            latest_check = FixityLog.objects.filter(package=self).order_by('-datetime_reported')[0]  # limit 1
        except IndexError:
            return None
        return latest_check.datetime_reported
```
Thoughts?